app/src/main/res/menu/inbox_sort.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Please reuse the code from #6210 instead of using a new copy of the menu item definitions.